test/test_autograd.py

```diff
@@ -855,6 +855,20 @@ def hook(grad):
             self.assertEqual(x.grad, expected_grad)
             self.assertIsNone(x_list[i].grad)
 
+    def test_grad_zero_grad_unused(self):
+        x = torch.tensor(0.5, dtype=torch.float32, requires_grad=True)
+        a = torch.tensor(1, dtype=torch.float32, requires_grad=True)
```

Comment:
nit: no need to specify dtype, dtype is float32 by default

Reply:
The dtypes were removed and `1` change to `1.0` to convert it to a float.